package.json

```diff
@@ -72,7 +72,7 @@
     "checksum": "cd build && find . -exec shasum -a256 {} ';' > ../checksums.txt && cd ..",
     "zip": "zip -ur bridge-$npm_package_version.zip README.md bridge-https.py checksums.txt && cd build && zip -ur ../bridge-$npm_package_version.zip ./* && cd ..",
     "release": "npm install && npm run build && npm run checksum && npm run zip",
-    "pilot:ganache": "ganache-cli --blockTime 1 --networkId 1 -m 'benefit crew supreme gesture quantum web media hazard theory mercy wing kitten' > /dev/null &",
+    "pilot:ganache": "ganache-cli --blockTime 1 --networkId 1 --host '0.0.0.0' -m 'benefit crew supreme gesture quantum web media hazard theory mercy wing kitten' > /dev/null &",
```

Comment:
What's this about?

Reply:
ah both this and the other change you pointed out are about allowing the eth nod eto be accessible from a mobile device on your wifi network. So you can go to `192.168.0.114:3000` or whatever and pull it up on your iphone. the eth node thing has been iffy, but works occasionally (?)

Reply:
Ah cool, that seems good! (: